react/components/TrapFocus.tsx

```diff
@@ -1,33 +1,47 @@
-import React, { useEffect, useRef, forwardRef } from 'react'
+import React, { useEffect, useRef } from 'react'
 
 import useForkRef from '../modules/useForkRef'
+import ownerDocument from '../modules/ownerDocument'
 
 interface Props {
   children: React.ReactElement
   open: boolean
 }
 
-const TrapFocus = forwardRef(function TrapFocus(props: Props, ref) {
+const TrapFocus = function TrapFocus(props: Props) {
   const { children, open } = props
-  const childRef = useRef<HTMLElement>(null)
-  const handleRef = useForkRef(childRef, ref)
+  const rootRef: React.MutableRefObject<HTMLElement | null> = useRef(null)
+  // I'm just passing rootRef here because I know that this will receive
+  // as children non class components that doesn't have forwaredRef
+  // if so, it wouldn't work properly
```

Comment:
I didn't understand this explanation very well. Also, in these cases, I prefer /* */ for comment so you can do multiline comments.

Reply:
Still needs fixing!